bundles/aliyun-bundle/build.gradle.kts

```diff
@@ -46,7 +46,10 @@ tasks.withType(ShadowJar::class.java) {
   // Relocate dependencies to avoid conflicts
   relocate("com.fasterxml.jackson", "org.apache.gravitino.aliyun.shaded.com.fasterxml.jackson")
   relocate("com.google", "org.apache.gravitino.aliyun.shaded.com.google")
-  relocate("com.sun", "org.apache.gravitino.aliyun.shaded.com.sun")
+  relocate("com.sun.activation", "org.apache.gravitino.aliyun.shaded.com.sun.activation")
+  relocate("com.sun.istack", "org.apache.gravitino.aliyun.shaded.com.sun.istack")
+  relocate("com.sun.jersey", "org.apache.gravitino.aliyun.shaded.com.sun.jersey")
+  relocate("com.sun.xml", "org.apache.gravitino.aliyun.shaded.com.sun.xml")
```

Comment:
So do you want to shade these jars, they're also mostly provided by jdk, right?

Reply:
they are not provided by jdk

Reply:
Which jars included these classes, AFAICK, they're provided by Sun as unsafe APIs.

Reply:
they are included by `hadoop-client-runtime-3.3.1.jar`.

We should note that these are third-party libraries that use the `com.sun.*` namespace but are not part of the JDK.

Reply:
I could not find any clear issues here. If you have tested it locally and with spark, we can merge it first.